website/www/site/content/en/documentation/dsls/dataframes/overview.md

```diff
@@ -115,3 +117,5 @@ pc1, pc2 = {'a': pc} | DataframeTransform(lambda a: expr1, expr2)
 
 [pydoc_dataframe_transform]: https://beam.apache.org/releases/pydoc/current/apache_beam.dataframe.transforms.html#apache_beam.dataframe.transforms.DataframeTransform
 [pydoc_sql_transform]: https://beam.apache.org/releases/pydoc/current/apache_beam.transforms.sql.html#apache_beam.transforms.sql.SqlTransform
+
+{{< button-colab url="https://colab.research.google.com/github/apache/beam/blob/master/examples/notebooks/tour-of-beam/dataframes.ipynb" >}}
```

Comment:
There's actually a TODO to link to this from the "Differences from pandas" page: https://github.com/apache/beam/blob/77fcf291d1986a8592f960e6ba77c77bcf9dc4f4/website/www/site/content/en/documentation/dsls/dataframes/differences-from-pandas.md#L90

Could you move it there?

Also it would be really nice if we could make it so the "Interactive Beam" section shows up on the website directly, in addition to a "run in colab" button, but I'm not sure if that's possible.

Reply:
Sure thing! I just added a button there as well.